### PR TITLE
Fix nonmatching functions throughout the repo

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/battle/anim/draw.c [hunk 1]
```diff
@@ -59,7 +59,6 @@ void sub_80D0C88(u8 taskId)
     task->func = sub_80D0D68;
 }
 
-#ifdef NONMATCHING
 void sub_80D0D68(u8 taskId)
```
Comment (camthesaxman) [inline src/battle/anim/draw.c]: Why were we unable to match this before?

Comment (PikalaxALT) [reply src/battle/anim/draw.c]: Couldn't tell you.

File: src/debug/matsuda_debug_menu.c [hunk 2]
```diff
@@ -460,14 +300,172 @@ static void sub_80A9F50(void)
...
     LoadFontDefaultPalette(&gWindowTemplate_81E6C3C);
 }
 
+//const u8 gMatsudaDebugMenu_UnknownByteArray[] = {0xFC, 0x02, 0x0F, 0x00, 0xFF};
+const u8 Unknown_083C922D[] = INCBIN_U8("graphics/unknown/ball.4bpp");
+const u16 Unknown_083C924E[] = INCBIN_U16("graphics/unknown/ball.gbapal");
+
+const u8 gUnknown_083C926E[][2] =
+    {
+        {2,  0},
```
Comment (camthesaxman) [inline src/debug/matsuda_debug_menu.c]: Why did you put extra indent spaces here?

Comment (PikalaxALT) [reply src/debug/matsuda_debug_menu.c]: no clue where that came from tbqh. might have been my IDE trying to do what it thought was cleaning it up?

File: src/roulette.c [hunk 3]
```diff
@@ -2236,386 +1818,77 @@ void sub_8116EF8(u8 r0)
...
-        case 0x2:
-            var3 = gSprites[eRoulette->var3C[0x7 + 0x1]].oam.paletteNum * 0x10;
-            break;
-        case 0x3:
-            var3 = gSprites[eRoulette->var3C[0x7 + 0x2]].oam.paletteNum * 0x10;
-            break;
-        case 0x4:
-            var3 = gSprites[eRoulette->var3C[0x7 + 0x3]].oam.paletteNum * 0x10;
-            break;
+            case 0x1:
```
Comment (camthesaxman) [inline src/roulette.c]: extra spaces

Comment (PikalaxALT) [reply src/roulette.c]: you know my position on indentation within switch contexts; i stand by it

**Pull request merged**